lib/bolt/transport/ssh/exec_connection.rb

```diff
@@ -12,8 +12,12 @@ def initialize(target)
           raise Bolt::ValidationError, "Target #{target.safe_name} does not have a host" unless target.host
 
           @target = target
-          ssh_config = Net::SSH::Config.for(target.host)
-          @user = @target.user || ssh_config[:user] || Etc.getlogin
+          begin
+            ssh_config = Net::SSH::Config.for(target.host)
+            @user = @target.user || ssh_config[:user] || Etc.getlogin
+          rescue => e
```

Comment:
```suggestion
          rescue StandardError
```
This will make rubocop happy (specifying an error class, and no assigning to a var without using it)

Reply:
Oh sorry no `=>`, just the `rescue StandardError`